src/commands/reminder/command.ts

```diff
@@ -25,21 +25,31 @@ const reminderLoop = () => {
 
   reminderCache = futureReminders;
 
-  pastReminders.forEach((r) => {
+  pastReminders.forEach(async (r) => {
```

Comment:
While this works, I don't think this is doing what you expect, awaiting in a foreach to read files in sequence

https://stackoverflow.com/a/37576787

Some of the answers in this thread provide good alternatives